lib/MenuDrawer.dart

```diff
@@ -59,6 +68,32 @@ class MenuDrawer extends StatelessWidget {
     return Divider(thickness: 1);
   }
 
+  Widget _createDrawerHeader(IconData icon, String text){
+    return UserAccountsDrawerHeader(
+      currentAccountPicture: new CircleAvatar(
+          backgroundColor: Colors.white,
+          //backgroundColor: Colors.deepPurple,
+          child: new Icon(
+            icon,
+            color: Colors.deepPurple[400],
+            //color:Colors.white,
```

Comment:
Remove comments.

Reply:
Comments contain alternative colors for user header.